extensions/wikia/EmbeddableDiscussions/styles/SettingsModal.scss

```diff
@@ -0,0 +1,106 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+
+.embeddable-discussions-settings-modal.modal {
```

Comment:
@latata I'm not sure if this is a convention for all of app, but this is the convention we used for the two recent extensions I've been involved in (CommunityPage and EmbeddableDiscussions). It is basically, <extension name>-<feature name>-<specific name>